test/e2e/test_tools.py

```diff
@@ -191,123 +194,161 @@ def test_get_project(self) -> None:
             )
 
     def test_validate_xml_against_schema(self) -> None:
-        self.assertTrue(validate_xml_against_schema(input_file=self.test_data_systematic_file))
+        subprocess.run(
+            f"poetry run dsp-tools xmlupload --validate-only --verbose {self.test_data_systematic_file.absolute()}",
+            check=True,
+            shell=True,
+            capture_output=True,
+            cwd=self.cwd,
+        )
 
     def test_xml_upload(self) -> None:
```

Comment:
Here, the diff is a bit confusing. In principle, it's the same than before, just expressed differently.

I split the method `test_xml_upload()` in two: 
The first uploads the the minimal data file (as before).
The second uploads the systematic data file (as before), then reads the mapping and calls the `id2iri` command, to make an incremental xmlupload (as before).